### PR TITLE
Remove hard coded timeline data source labels.

### PR DESCRIPTION
**Pull request by ian-leggett**
Previously the company source labels were hard coded. These are now available in the backend and are being pulled through.

timeline

### CODE REVIEW

File: test/unit/data/companies/timeline.json [hunk 1]
```diff
@@ -4,27 +4,27 @@
   "previous": null,
   "results": [
     {
-      "data_source": "companies_house.companies",
```
Comment (elcct) [inline test/unit/data/companies/timeline.json]: The backend also sends `data_source` field, so I would keep that in the test.

Comment (ian-leggett) [reply test/unit/data/companies/timeline.json]: Just adding this back to the JSON.

**Comment by yeahfro** [comment]
@Ian, the commit messages should read like an instruction, ie "Remove", "Update", "Replace"

Also, as we are going through the PR process we use fixups and then once approved, the fixups are squashed

**Comment by ian-leggett** [comment]
@yeahfro just updated the commit comments to "Remove", "Update", "Replace", I was trying to --fixup one of the commits for squashing but it will merge into the commit for the test update rather than the one it relates too.

**Comment by yeahfro** [comment]
I am on the cusp of signing this off but the commits don't really make sense as you would apply the first and the test would fail until the second is applied.

**Pull request merged**